README.md

```diff
@@ -102,7 +102,6 @@ Execute command in the Project Root folder
     @pmm-ami                legacy/deprecated
     @not-ui-pipeline        legacy/deprecated
     @not-pr-pipeline        legacy/deprecated
-    @cli                    cli related tests
```

Comment:
Why remove this? We use this tag.

Reply:
Fixed, it was conflict after changing branch from 2.29.1 to main